launcher/src/test/java/org/apache/spark/launcher/SparkSubmitCommandBuilderSuite.java

```diff
@@ -59,6 +59,8 @@ public void testClusterCmdBuilder() throws Exception {
   @Test
   public void testCliParser() throws Exception {
     List<String> sparkSubmitArgs = Arrays.asList(
+      parser.CLASS,
```

Comment:
No they are not necessary and just left while I fixing the broken tests. I will revert these.